### PR TITLE
[governance change; request for comments] Add emphathy to guiding principles

### LINKED ISSUE #13: some commit messages from contributors are uninformative and so they slow the entire team down when identifying a changeset
- sometimes contributors call their commit messages "fix" and then it contains actually an entire changeset for important fixes
- it's important to us to identify commits by their message as we have to sometimes look something up or revert a thing
- so it'd be ideal if commit messages would always contain a nice summary of what's happening within the changeset
- but we can also not ask a CI to validate commit messages as a machine can't tell if a commit message is a nice description or some blank-filler statement

### PR DESCRIPTION
**Pull request by TimDaub**
- Fixes #13

### CODE REVIEW

**Comment by djfnd** [comment]
I don't understand what the suggested change is here. Empathy in terms of language choice? Or is there a specific set of suggested wording changes?

**Comment by djfnd** [comment]
Ignore that comment. Approved from my side.

File: social-architecture.md [hunk 1]
```diff
@@ -110,7 +110,16 @@ All contributors shall prioritize automation over chores. Implementations of
...
 
-### Safety
+#### Empathy
+
+All contributors actively attempt to comprehend their peers thinking and
+backgrounds during design and execution processes.
+
+Empathic design is a reflection of our blind spots and strengths without an
+entitlement feeling understood. Instrad acknowledging complexity and
+responsibility of effectively communicating.
```
Comment (il3ven) [inline social-architecture.md:120]:
```suggestion
Empathic design is a reflection of our blind spots and strengths without an
entitlement feeling understood. Instead acknowledging complexity and
responsibility of effectively communicating.
```

I guess you meant "instead".

**Review by neatonk — APPROVED** [review]
Looks good to me.

File: social-architecture.md [hunk 2]
```diff
@@ -110,7 +110,16 @@ All contributors shall prioritize automation over chores. Implementations of
 automation must be reproducible, sharable between contributors and (ideally)
 tested.
 
-### Safety
+#### Empathy
+
+All contributors actively attempt to comprehend their peers thinking and
+backgrounds during design and execution processes.
+
+Empathic design is a reflection of our blind spots and strengths without an
```
[the remark this answers is not in the document]
Comment (TimDaub) [reply social-architecture.md:118]: Ok makes sense. Yeah I meant empathic design as in: e.g. we write informative commit messages etc as this makes empathically sense to us given that we know others would struggle more to identify a change set if it was without good commit message.

Comment (TimDaub) [inline social-architecture.md:118]:
```suggestion
Empathic contributing is a reflection of our blind spots and strengths without an
```

**Pull request merged**